_posts/2022-08-11-temp-title/my-4-biggest-internship-takeaways.Rmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "My 4 Biggest Internship Takeaways"
+description: |
+  What I've learned after interning at the PCCTC.
+author:
+  - name: Kirina Sirohi
+    url: {}
+base_url: https://www.pipinghotdata.com/
+date: 2022-08-19
+preview: cover_image.png
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+# Introduction
```

Comment:
I think you also need more about you & the internship to add context

- name
- college & year
- duration of internship (8 weeks)
- remote work environment
- one sentence about scope of work (medical monitoring on clinical trials?)
- one sentence about technical skills learned